main.nf

```diff
@@ -330,6 +359,14 @@ process miRBasePostAlignment {
 }
 
 
+def rule_step6(file){
```

Comment:
Maybe a nicer name again as above.

Could make an argument to try to use a single function for both, but it's not many lines and I think this is clearer 👍